### ISSUE TITLE
`ag-cosmos-helper status` should write to stdout, not stderr

### ISSUE THREAD

**Issue by warner** [Agoric/agoric-sdk#2888]
## Describe the bug

I used `ag-cosmos-helper status` for the first time today. I was surprised that the machine-parseable JSON output is written to stderr, rather than stdout. It's a drag to require a redirect to parse the output.

Instead of:

```sh
% ag-cosmos-helper status 2>&1 | jq .SyncInfo
```

I'd prefer to run:

```sh
% ag-cosmos-helper status | jq .SyncInfo
```

If we can fix this, we could also update the guides at https://github.com/Agoric/agoric-sdk/wiki/Validator-Guide-for-Devnet#syncing-your-node and https://github.com/Agoric/agoric-sdk/wiki/Validator-Guide-for-Incentivized-Testnet#syncing-your-node to remove the `2>&1` bit.

**Comment by michaelfig** [comment]
We're working on it in cosmos/cosmos-sdk#8628 which should be backported to cosmos-sdk v0.42.x.

I don't know if there's value added by keeping the current issue open.  Tracking, I guess.

**Comment by michaelfig** [comment]
Solved in [cosmos-sdk v0.43.0-alpha1](https://github.com/cosmos/cosmos-sdk/tree/v0.43.0-alpha1).  We will be adopting the 0.43.0 release sooner or later.

**Comment by michaelfig** [comment]
Done!
